### PR TITLE
Edit functionality

### PR DESCRIPTION
**Pull request by pmdredd**
This PR adds the ability to edit the details of an existing record (e.g. Student, Coursework) via a new button in the details view. The button takes you to a new form that is pre-populated with the current records's values. On submission, it does the usual error checking and then updates that record.

Note that because PR #22 has not been merged yet, the related id option is not a nice dropdown. I think once it is merged the functionality in this PR is a good candidate for a TDD refactor (e.g. refactor to use the nice dropdown and also to separate the update functionality into a dedicated function)

This PR completes user story "Edit Coursework" (and also adds this functionality to edit Students, Courses and Submissions).

### CODE REVIEW

File: courses/edit_course_form.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+require_once 'course_functions.php';
+require_once '../header.html';
+
+$course = getCourseById($_GET['id']);
+
```
Comment (Lairdd1989) [inline courses/edit_course_form.php]:
```suggestion
```

File: courses/edit_course_form.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+?>
+
+
```
Comment (Lairdd1989) [inline courses/edit_course_form.php]:
```suggestion
```

File: courseworks/edit_coursework_form.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?php
+require_once 'coursework_functions.php';
+require_once '../header.html';
+
+$coursework = getCourseworkById($_GET['id']);
+
```
Comment (Lairdd1989) [inline courseworks/edit_coursework_form.php]:
```suggestion
```

File: students/edit_student_form.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?php
+require_once 'student_functions.php';
+require_once '../header.html';
+
+$student = getStudentById($_GET['id']);
+
+?>
+
```
Comment (Lairdd1989) [inline students/edit_student_form.php]:
```suggestion
```

File: submissions/edit_submission_form.php [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<?php
+require_once 'submission_functions.php';
+require_once '../header.html';
+
+$submission = getSubmissionById($_GET['id']);
+
```
Comment (Lairdd1989) [inline submissions/edit_submission_form.php]:
```suggestion
```

**Comment by pmdredd** [comment]
> 
> 
> Great work added some comments on formatting for necessary blank lines.

Cheers, I have commited your changes. If everyone else is happy with it then I'll go ahead and merge.

**Pull request merged**